components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -458,6 +482,18 @@ func (r *Reconciler) DeletePublisherProxySecret(ctx context.Context) error {
 		Namespace: deployment.PublisherNamespace,
 		Name:      deployment.PublisherName,
 	}
+	return r.deleteSecret(ctx, secretNamespacedName)
+}
+
+func (r *Reconciler) DeleteNATSSecret(ctx context.Context) error {
```

Comment:
do we really need to export such functions?

Reply:
no. I will make them internal 👍